app/src/ui/index.tsx

```diff
@@ -99,15 +103,94 @@ process.once('uncaughtException', (error: Error) => {
       `An uncaught exception was thrown. If this were a production build it would be reported to Central. Instead, maybe give it a lil lookyloo.`
     )
   } else {
-    sendErrorReport(error, {
+    const extra: Record<string, string> = {
       osVersion: getOS(),
       guid: getGUID(),
-    })
+    }
+
+    try {
+      if (currentState) {
+        if (currentState.currentBanner !== null) {
+          extra.currentBanner = currentState.currentBanner.type
```

Comment:
:art: Should we stringify this value like the others?

```suggestion
          extra.currentBanner = `${currentState.currentBanner.type}`
```

Reply:
This value is actually already a string. Anything else would have caused a compile error since `extra` is `Record<string, string>`

Reply:
Nevermind: https://github.com/desktop/desktop/pull/7693#discussion_r290852181